diff_settings.py

```diff
@@ -3,3 +3,8 @@ def apply(config, args):
     config["myimg"] = "source.bin"
     config["mapfile"] = "build.map"
     config["source_directories"] = ["."]
+    #config["build_dir"] = "build/"
```

Comment:
rename to mw_build_dir and/or add a comment that it's mw only, to decrease confusion about this being necessary, perhaps?